msteams-platform/concepts/capabilities-overview.md

```diff
@@ -1,16 +1,21 @@
 ---
-title: Understanding Teams app capabilities
+title: Understand Teams app capabilities
```

Comment:
```suggestion
title: Understand app capabilities
```
The `tltle` metadata property is automatically appended with "- Teams" during publishing, so you don't have to repeat it here manually.